### ISSUE TITLE
Feature:  Comment field

### ISSUE THREAD

**Issue by craigerl** [nwdigitalradio/n7nix#9]
rmsgw_aci  doesn't currently provide a way to post the "comment" for the station.  I've tried and failed to add it.

**Comment by n7nix** [comment]
Functionally rmsgw_aci sends the 'automatic Check-in' information to Winlink services. That includes the channels information every 2 hours found in the channels.xml file and the software version every 24 hours found in the .version_info file. The station "Comments" is maintained in the sysops.xml file and is not used by rmsgw_aci. 

To change sysops.xml file please read [this link](https://github.com/nwdigitalradio/rmsgw/blob/master/admin/README.md) which describes how to use updatesysop.py, getsysop.py & mksysop.py python scripts.

You can read what is currently in your sysop record by running getsysop.py script found in /etc/rmsgw

**Comment by craigerl** [comment]
Great - thanks, it looks like winlink updated their api, including a key/token,

python2 updatesysop.py  -d

```
svc_url = https://api.winlink.org/status/sysop/add?format=json
Response = 404
404
Request status code: 404
Debug: Response = 404
Debug: Content type: text/plain; charset=utf-8
```


I'll see what I can do.

**Comment by n7nix** [comment]
> Great - thanks, it looks like winlink updated their api, including a key/token,

Yes. You need to update your Linux RMS Gateway source then follow the instructions on the previous link I sent
This will make your Linux RMS  GW compatible with V5 of Winlink Web Services

All 3 of these worked for me.
```bash
# ./updatesysop.py
('Posting sysop record update for', 'N7NIX', '...')
Found Comments: [34] V5 Services Test: 2021-06-08 14:35
Sysop update for N7NIX successful.

# python2 updatesysop.py
('Posting sysop record update for', 'N7NIX', '...')
Found Comments: [34] V5 Services Test: 2021-06-08 14:35
Sysop update for N7NIX successful.

# python3 updatesysop.py
Posting sysop record update for N7NIX ...
Found Comments: [34] V5 Services Test: 2021-06-08 14:35
Sysop update for N7NIX successful.
```
